<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-17T04:31:47Z" name="LoginSuite" finished-at="2022-01-17T04:31:52Z" duration-ms="5002">
    <groups>
    </groups>
    <test started-at="2022-01-17T04:31:47Z" name="Test123" finished-at="2022-01-17T04:31:52Z" duration-ms="5002">
      <class name="test.NewLoginTest">
        <test-method is-config="true" signature="initiate()[pri:0, instance:test.NewLoginTest@17f62e33]" started-at="2022-01-17T04:31:47Z" name="initiate" finished-at="2022-01-17T04:31:52Z" duration-ms="4522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initiate -->
        <test-method signature="test1()[pri:0, instance:test.NewLoginTest@17f62e33]" started-at="2022-01-17T04:31:52Z" name="test1" finished-at="2022-01-17T04:31:52Z" duration-ms="41" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method signature="test2()[pri:0, instance:test.NewLoginTest@17f62e33]" started-at="2022-01-17T04:31:52Z" name="test2" finished-at="2022-01-17T04:31:52Z" duration-ms="17" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method signature="test3()[pri:0, instance:test.NewLoginTest@17f62e33]" started-at="2022-01-17T04:31:52Z" name="test3" finished-at="2022-01-17T04:31:52Z" duration-ms="9" status="FAIL">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/commons/io/output/UnsynchronizedByteArrayOutputStream]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/commons/io/output/UnsynchronizedByteArrayOutputStream
at org.apache.poi.poifs.filesystem.FileMagic.valueOf(FileMagic.java:209)
at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:206)
at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:172)
at util_main.TestUtile.ExcelRead(TestUtile.java:39)
at facebook_main.Login_new.verylogin(Login_new.java:46)
at test.NewLoginTest.test3(NewLoginTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.io.output.UnsynchronizedByteArrayOutputStream
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:636)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
... 31 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method is-config="true" signature="end()[pri:0, instance:test.NewLoginTest@17f62e33]" started-at="2022-01-17T04:31:52Z" name="end" finished-at="2022-01-17T04:31:52Z" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- test.NewLoginTest -->
    </test> <!-- Test123 -->
  </suite> <!-- LoginSuite -->
</testng-results>
